src/dapps/ManageENS/ABI/registrarAbi.js

```diff
@@ -23,30 +23,6 @@ export default [
     payable: false,
```

Comment:
I think we can safely delete this whole file? are we referencing this anywhere?

Reply:
@kvhnuke It does look like its been referenced on ManageENS.vue.341